File: Day025/Day25.java
package UselessJava.Day025;

import java.util.Scanner;

public class Day25 {
    public String generateRandomCoordinate(){
        char[] lat = new char[]{'N', 'S'};
        char[] lon = new char[]{'E', 'W'};
        double latitude = Math.random() * 90;
        double longitude = Math.random() * 180;
        
        String latString =  toDMS(latitude) + lat[(int)(Math.random() * 2)];
        String longString = toDMS(longitude) + lon[(int)(Math.random() * 2)];
        return latString + " " + longString;
    }
    
    private String toDMS(double decimal){
        double d = decimal;
        long l1 = (long) d;
        d -= l1;
        d *= 60;
        long l2 = (long)d;
        d -= l2;
        d *= 60;
        long l3 = (long)d;
        return l1 + "º " + l2 + "' " + l3 + "\"";
    }
}

class Main{
    public static void main(String[] args) {
        Day25 d = new Day25();
        Scanner sc = new Scanner(System.in);
        while(true){
            System.out.println("press enter to generate a random coordinate!");
            String s = sc.nextLine();
            System.out.println(d.generateRandomCoordinate());
        }
    }
}

File: days/Day025/web/Day25.js
function foo (){
    const lat = ['N', 'S'];
    const lon = ['E', 'W'];
    const latitude = Math.random * 90;
    const longitude = Math.random * 180;

    const latString = toDMS(latitude) + lat[Math.round(Math.random() * 2)];
    const longString = toDMS(longitude) + lon[Math.round(Math.random() * 2)];
    return latString + " " + longString; //JS isn't that hard i guess. It just takes a lot of google sarch
}

function toDMS(bar){
    var d = bar;
    const l1 = Math.round(d);
    d -= l1;
    d *= 60;
    const l2 = Math.round(d);
    d -= l2;
    d *= 60;
    const l3 = Math.round(d);
    return l1.toString + "º " + l2.toString + "\' " + l3.toString + "\"";
}